src/main/java/controller/deleteServlet.java

```diff
@@ -0,0 +1,45 @@
+package controller;
+
+import util.DBConnection;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.Statement;
+
+    @WebServlet("/delete")
+    public class deleteServlet  extends HttpServlet {
+        @Override
+        protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+            HttpSession session = req.getSession();
+            int partyId = (int) session.getAttribute("partyId");
+            System.out.println(partyId);
+            if (session.getAttribute("username")!=null){
+                partyId = (int) session.getAttribute("partyId");
```

Comment:
Please don't put everything in session and read it from the session. This is a bad practice.